docs/src/main/sphinx/security/inspect-pem.rst

```diff
@@ -0,0 +1,137 @@
+.. _pem_certs:
+
+=========
+PEM files
+=========
+
+PEM (Privacy Enhanced Mail) is a standard for public key and certificate
+information, and an encoding standard used to transmit keys and certificates.
+The PEM format is specified in the :ref:`RFC 5958 <glossPEM>` standard.
```

Comment:
I don't think we need a link to the specification... PEM is really an envelope format that people use for lots of things, so the spec isn't super helpful.

Reply:
Delete my scholarly updating of the reference to PKCS #8, which is now listed as obsolete and replaced by RFC 5958? Well, OK.